packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -149,7 +154,10 @@ export class WaferMap extends FoundationElement {
                 return;
             }
             const { height, width } = entry.contentRect;
-            this.canvasSideLength = Math.min(height, width);
+            this.canvas.width = width;
+            this.canvas.height = height;
```

Comment:
Can you include the comment example given from our discussion, i.e.
```suggestion
            // Updating the canvas size clears its contents so update it explicitly instead of
            // via template bindings so we can confirm that it happens before render
            this.canvas.width = width;
            this.canvas.height = height;
```

Reply:
sure